doc/workshop/exercises/analysis/dtm/dtm.rst

```diff
@@ -148,6 +148,8 @@ Notes
 
 2. :ref:`writers.gdal` can be used for large data, but it does not interpolate
    a typical `TIN`_ surface model.
+   
+3. If you are missing the Raster tab, check your plugins in QGIS. If you have none you may need to reinstall through conda and make sure you have all the packages.
```

Comment:
What is this about?

Reply:
If you don't install from online or make sure you get the plugins in conda you can't use Rasters